src/OrchardCore.Modules/OrchardCore.ContentFields/Assets/js/vue-multiselect-userpicker.js

```diff
@@ -0,0 +1,92 @@
+function debounceUserPicker(func, wait, immediate) {
```

Comment:
For another PR, but maybe we could make this guy even more generic so we would not need to copy it around.